compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -1926,6 +1960,9 @@ void endExternalFunctionBody(int annotCount) {
         // reversing the collected annotations to preserve the original order
         Collections.reverse(tempAnnotAttachments);
         tempAnnotAttachments.forEach(invokableNode::addExternalAnnotationAttachment);
+
+        FunctionBodyNode body = this.funcBodyNodeStack.pop();
+        invokableNode.setBody(body);
```

Comment:
move the extern function related logic to the extern func body from the invoke node.